Borba02/nss_Prop_B_Gone
# added restore button and installed MUI to prepare for styling

#### Changes Made
1. added a button to appear on /history to restore entries back into the other views
1. installed MUI to prepare for styling

#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout bb-restoreButton
    ```
2. Open a new Terminal tab and navigate to the server directory.
3. Test app functionality.

- [ ]  Go to /history
- [ ]  Click on Restore button
- [ ] Ensure entry has been removed from /history and disposedOf has been set to an empty string on the database

4. View code file.
    > Confirm file modifications are present as indicated above.
